ludwig/features/numerical_feature.py

```diff
@@ -357,15 +357,17 @@ class Log1pTransformer:
     def __init__(self, **kwargs: dict):
         pass
 
-    def transform(self, x: np.ndarray) -> np.ndarray:
+    @staticmethod
```

Comment:
I wouldn't make those methods static even if they don't need to use self, because we are calling them as `instance.transform(x)`, so having `self` as the first parameter of the function captures the `instance` variable.

Reply:
done.